calico/felix/test/test_config.py

```diff
@@ -660,3 +664,14 @@ def test_drop_valid(self):
             }
             config = load_config("felix_missing.cfg", host_dict=cfg_dict)
             self.assertEqual(config.ACTION_ON_DROP, value)
+
+    def test_interface_prefix(self):
+        cfg_dict = {"InterfacePrefix": "foo"}
+        config = load_config("felix_interface_prefix.cfg",
+                             host_dict=cfg_dict)
+        self.assertEqual(config.IFACE_PREFIX, ['foo'])
+
+        cfg_dict = {"InterfacePrefix": "foo,bar"}
+        config = load_config("felix_interface_prefix.cfg",
+                             host_dict=cfg_dict)
+        self.assertEqual(config.IFACE_PREFIX, ['foo', 'bar'])
```

Comment:
Sure, good idea.  When you say 'as a tox chain', is that the same as adding a tox environment?  (I believe that OpenStack projects do this with a tox 'pep8' environment.)  Would you like to propose a change for that?